apps/src/p5lab/spritelab/commands/worldCommands.js

```diff
@@ -40,6 +40,14 @@ export const commands = {
     audioCommands.playSound({url, loop: false});
   },
 
+  playSpeech(speech) {
+    audioCommands.playSpeech({
+      text: speech,
+      gender: 'female',
```

Comment:
hardcoded gender + language for now, since this is just for Mike and Amy to prototype with. Eventually, we'll probably want to either add dropdowns to this block or make another block to set these.